Build.cmake

```diff
@@ -69,9 +69,9 @@ ENDIF()
    #MESSAGE(" HEADER FILES ARE: ${HEADER_FILES}")
 
    IF (MSVC OR MINGW) 
-         list(REMOVE_ITEM SRC_FILES  ${LOG_SRC}/g3log/crashhandler_unix.cpp)
-   ELSE()     
-         list(REMOVE_ITEM SRC_FILES  ${LOG_SRC}/crashhandler_windows.cpp ${LOG_SRC}/g3log/stacktrace_windows.hpp ${LOG_SRC}/stacktrace_windows.cpp)
+         list(REMOVE_ITEM SRC_FILES  ${LOG_SRC}/crashhandler_unix.cpp)
+   ELSE()
+         list(REMOVE_ITEM SRC_FILES  ${LOG_SRC}/crashhandler_windows.cpp ${LOG_SRC}/stacktrace_windows.hpp ${LOG_SRC}/stacktrace_windows.cpp)
```

Comment:
Thanks for the pull request. 
stacktrace_windows.hpp  needs to be under 
```
${LOG_SRC}/g3log/stacktrace_windows.hpp
```

The software is located as follows
```
g3log
       / src
             /  *.cpp   *.ipp
            / g3log
                                 /*.hpp

```